recruitment/introduction.md

```diff
@@ -0,0 +1,26 @@
+## Holiday Extras
+
+#### Why a technical task?
+
+You might be wondering why we feel its important to test candidates. At Holiday Extras we use a variety of different technologies and we don't expect you to be aware of all of them! We've designed both of our tasks with this in mind.
+
+Below is a few points that we tend to look for when reviewing your submission:
+
+- Quality (is it functional?)
+- Structure (code layout, abstraction, modularisation)
+- Usability and design
+
+#### The tasks
+
+- [Build an API](developer-API-task.md)
+- [Flickr API](developer-flickr-task.md)
+
+If you have _not_ been asked to complete a specific task in the email you received from our recruitment team then we're leaving it up to you! Feel free to pick whichever task you believe would best showcase your skills.
+
+Alternatively if you regularly contribute to open-source and believe that we will be able to gain a valuable insight into how you work, then please pass on these details to our recruitment team at: <recruitment@holidayextras.com>.
+
+#### What to do after completing the task(s)?
+
+When you feel you've completed the task we would appreciate it if you could publish it online so we are in a position to be able to review it. We recommend that you upload your code to a Github/Bitbucket/Gitlab repository, however it's fine if you'd rather send it as an attachment in an email to <recruitment@holidayextras.com>.
```

Comment:
I would get them to email me directly, rather than recruitment. A bit more personal, and it will be me who deals with it anyway :)